modules/apache-airflow/templates/values.yaml

```diff
@@ -2446,12 +2445,12 @@ cleanup:
   labels: {}
 
   resources:
-   limits:
-    cpu: 1000m
-    memory: 512Mi
-   requests:
-    cpu: 100m
-    memory: 128Mi
+  #  limits:
```

Comment:
For spots where it is `enabled: false`, no changes need to be made as it won't have any affect on what we have deployed.